PostHog/PostHogSDK.swift

```diff
@@ -219,7 +219,9 @@ let maxRetryDelay = 30.0
         return props
     }
 
-    @objc public func register(_ properties: [String: Any]) {
+    // register is a reserved word in ObjC
+    @objc(registerProperty:)
```

Comment:
Would it not be plural? 
```suggestion
    @objc(registerProperties:)
```

Reply:
Good point